### PR TITLE
Copy favicons from microsite

### PR DESCRIPTION
**Pull request by burgwyn**
## Update Favicon

Addresses #1071 with assets from the [microsite](https://github.com/spotify/backstage-microsite/pull/163)

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by stefanalund** [comment]
Thanks for the PR! I think there is something missing, because I don't see the favicon in my browser tab:

Screenshot 2020-06-03 at 09 45 32

File: packages/app/public/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<svg id="Assets" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 337.46 428.5">
+    <title>
+        favicon
+    </title>
+    <style>
+        path {
```
Comment (shmidt-i) [inline packages/app/public/favicon.svg:6]: This is going to affect **all** `path` elements in the document, we don't want that. Instead, use `withStyles` from the `@material-ui/core` and also then you are able to read `theme.palette.type` which would be `light` or `dark` based on the current theme used.

**Review by shmidt-i — CHANGES_REQUESTED** [review]

File: packages/app/public/favicon.svg (hunk 1 above)
Comment (shmidt-i) [reply packages/app/public/favicon.svg:6]: Sorry, misread the context, disregard the material-ui part. Anyway we need to make this more modular, at least like a className

**Comment by stefanalund** [comment]
Rebased on `master` and now we are getting closer:

Screenshot 2020-06-08 at 20 23 06
Screenshot 2020-06-08 at 20 23 14

I believe the SVG version of the favicon is not used because of the same issue as https://github.com/spotify/backstage/issues/1202.

**Review by freben — APPROVED** [review]
Let's merge and tweak! Thank you for your contribution!

**Review by shmidt-i — APPROVED** [review]
Let's fix it in a separate PR!

**Pull request merged**